examples/plotting/masked_composite_plot.py

```diff
@@ -0,0 +1,57 @@
+"""
+================================
+Creating a Masked Composite Plot
+================================
+
+This example combines creating a composite plot with masking out the
+solar disk. The purpose is to introduce users to the plotting process
+required to overlay multiple Maps that have been modified with other
+**sunpy** functions. The resulting plot in this tutorial shows information
+on the upper photosphere, quiet corona, and magnetogram contours.
```

Comment:
👍 really like describing the different solar layers being demonstrated in the example